node/builder_chain.go

```diff
@@ -48,9 +48,16 @@ var ChainNode = Options(
 	// Full node or lite node
 	// TODO: Fix offline mode
 
+	// FVM: builtin actor bundle loading
+	//      Note: this has to load before the upgrade schedule, so that we can patch in the
+	//      right manifest cid.
+	//      This restriction will be lifted once we have the final actors v8 bundle and we know
+	//      the manifest cid.
+	Override(BuiltinActorsKey, modules.LoadBultinActors),
```

Comment:
This can now be (even should)
```suggestion
	Override(new(dtypes.BuiltinActorsLoaded), modules.LoadBultinActors),
```

Reply:
yrah, no need for the special.